src/test/java/pl/marekjava/PasswordManagerTest.java

```diff
@@ -0,0 +1,30 @@
+package pl.marekjava;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class PasswordManagerTest {
+    @Test
+    public void isGenerateRandomPassword() {
+        PasswordManager pm = new PasswordManager();
+        List<String> list = new ArrayList<>();
+        list.add("Koło fortuny");
+        pm.setPasswords(list);
+        String randomPassword = pm.getRandomPassword();
+        assertTrue(randomPassword.equals("Koło fortuny"));
+    }
+
+    @Test
+    public void throwExceptionWhenIsNoUniquePasswords() {
+        Assertions.assertThrows(IllegalStateException.class, () -> {
+            PasswordManager pm = new PasswordManager();
+            for (int i = 0; i <= pm.getSizeOfPasswords(); i++) {
+                pm.getRandomPassword();
+            }
+        });
+    }
```

Comment:
Tu również dodałbym kolejny przypadek testowy - bo teraz sprawdzamy czy poleci illegalState z pustej listy haseł, co jest ok, ale warto też sprawdzić czy poleci jeśli dodamy dwa hasła, a wylosujemy trzykrotnie.

Reply:
Ale tutaj właśnie iteruję po całej liście haseł, dostępnej z konstruktora PasswordManagera od 0 do rozmiaru listy włącznie. Czyli dodanych jest 6 haseł a losowanych jest 7. Najwyżej dodam jeszcze dla pustej listy czy poleci wyjątek.

Reply:
A słusznie, zapomniałem, że w konstruktorze domyślnie tworzysz zestaw haseł. To nie ma tematu :)